spec/rubocop/cop/style/negated_if_else_condition_spec.rb

```diff
@@ -323,7 +323,7 @@
     RUBY
   end
 
-  it 'does not register an offense when not whe whole condition is negated' do
+  it 'does not register an offense when not the whole condition is negated' do
```

Comment:
```suggestion
  it 'does not register an offense when the whole condition is negated' do
```

I'm a bit confused about this test statement though because the whole condition is not negated...

Reply:
I get it. I've updated this with your suggestion. Thank you!

Reply:
Maybe `it 'does not register an offense when only part of the condition is negated'` would be clearer.